packages/react-dom/src/events/DOMLegacyEventPluginSystem.js

```diff
@@ -191,9 +191,10 @@ function extractPluginEvents(
   eventSystemFlags: EventSystemFlags,
 ): Array<ReactSyntheticEvent> | ReactSyntheticEvent | null {
   let events = null;
-  for (let i = 0; i < plugins.length; i++) {
+  const legacyPlugins = ((plugins: any): Array<LegacyPluginModule<Event>>);
```

Comment:
Why the any?

Reply:
We're casting it here. It's not an `any`.

Reply:
Yeah but if you need the cast to go through `any`, it's not sound.

Reply:
Yeah, it's difficult to type here though as it depends on the feature flag depending on what the actual type will be. I don't know any other way of casting this to what we want, let me know if there is that you know of?